package.json

```diff
@@ -8,6 +8,21 @@
   "main": "dist/a11y-dialog.js",
   "module": "dist/a11y-dialog.esm.js",
   "types": "dist/a11y-dialog.d.ts",
+  "exports": {
+    ".": {
```

Comment:
When you do `import A11yDialog from 'a11y-dialog'` you get `./dist/a11y-dialog.esm.js`, and when you do `const A11yDialog = require('a11y-dialog)` you get `./dist/a11y-dialog.js`